models/nc_news.model.js

```diff
@@ -33,4 +33,42 @@ const getArticles = () => {
       return data.rows;
     });
 };
-module.exports = { selectTopics, getEndpoints, selectArticleById, getArticles };
+
+const getComment = (articleId) => {
+  return db
+    .query(
+      `SELECT * FROM comments 
+  LEFT JOIN articles
+   ON comments.article_id = articles.article_id
+   WHERE comments.article_id = $1
+   ORDER BY comments.created_at DESC`,
+      [articleId]
+    )
+    .then((data) => {
+      if (data.rows[0] === undefined) {
```

Comment:
In this case, we need to check why we are returning an empty row, as this could be due to one of two reasons:
article exists but has no comments
OR
article does not exist
You will need to check whether the article exists in order to determine this, then handling it accordingly. Ideally if it has no comments but does exist, we want an empty array back. Otherwise if it doesn't exist, we want an error 😊